powerauth-load-tests/pom.xml

```diff
@@ -64,18 +63,42 @@
             <version>${gatling.version}</version>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>io.gatling</groupId>
+            <artifactId>gatling-test-framework</artifactId>
+            <version>${gatling.version}</version>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.postgresql</groupId>
+            <artifactId>postgresql</artifactId>
+            <version>42.7.2</version>
```

Comment:
May we use the implicit postgres version or is there any reason why we need this one specifically?